tests/440-box-plugin.ts

```diff
@@ -77,21 +77,71 @@ Scenario(
   },
 )
 
-// TODO: Fails but should work imo. Fix warning message implementation.
-// Scenario.only('Empty box warning message appears when box content is empty', async ({ I }) => {
-//   I.amOnPage('/entity/create/Article/1377')
-//   addBoxPlugin(I, 'blank')
-
-//   I.say('Empty warning message visible after creating new box')
-//   I.seeElement('$plugin-box-empty-content-warning')
-
-//   I.say('Empty warning message visible even if title is not empty')
-//   I.click('$plugin-text-editor', '$plugin-box-title')
-//   I.type('Boxtitel')
-//   I.seeElement('$plugin-box-empty-content-warning')
-
-//   I.say('Empty warning message not visible if content not empty')
-//   I.click('$plugin-text-editor', '$plugin-box-content')
-//   I.type('Boxinhalt')
-//   I.dontSeeElement('$plugin-box-empty-content-warning')
-// })
+Scenario('Empty box warning message appears when box content is empty', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+  addBoxPlugin(I, 'blank')
+
+  I.say('Empty warning message visible after creating new box')
+  I.click('input[placeholder="Titel"]') // unfocus box plugin
+  I.seeElement('$plugin-box-empty-content-warning')
+
+  I.say('Empty warning message visible even if title is not empty')
+  I.click('$plugin-text-editor', '$plugin-box-title')
+  I.type('Boxtitel')
+  I.click('input[placeholder="Titel"]') // unfocus box plugin
+  I.seeElement('$plugin-box-empty-content-warning')
+
+  I.say('Empty warning message not visible if content not empty')
+  I.click('$plugin-text-editor', '$plugin-box-content')
+  I.type('Boxinhalt')
+  I.click('input[placeholder="Titel"]') // unfocus box plugin
+  I.dontSeeElement('$plugin-box-empty-content-warning')
+})
+
+Scenario('Set box type using keyboard', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+  
+  I.say('Create box plugin')
+  I.click('$plugin-text-editor', '$plugin-article-content')
+  I.type('/')
+  I.type('Box')
+  I.pressKey('Enter')
+  I.seeElement('$plugin-box-initial-type-chooser')
+
+  I.say('Use the Tab key to select a type')
+  I.pressKey('Tab')
+  I.pressKey('Enter')
+  I.seeElement('$plugin-box')
+})
+
+
+Scenario('Change box type using keyboard', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+  addBoxPlugin(I, 'blank')
+
+  I.say('Pick a new box type by only using keys')
+  I.pressKey('Tab')
+  I.pressKey('Space')
+  I.pressKey('ArrowDown')
+  I.pressKey('Enter')
+  I.see('Beispiel')
```

Comment:
Searching for "Beispiel" across the whole page could lead to false positives if "Beispiel" appears somewhere else on the page. But something like this should make it more robust: 
```suggestion
  I.see('Beispiel', '$plugin-box')
```

Reply:
Unfortunately, even this is not specific enough and gives false positives.

This works: `I.see('Beispiel', locate('span.text-brand').inside('$plugin-box'))`